Doc/library/urllib.parse.rst

```diff
@@ -575,6 +575,16 @@ task isn't already covered by the URL parsing functions above.
    Example: ``unquote_to_bytes('a%26%EF')`` yields ``b'a&\xef'``.
 
 
+.. function:: unquote_to_bytes_plus(string)
+
+   Like :func:`unquote_to_bytes`, but also replace plus signs by spaces, as required for
+   unquoting HTML form values.
```

Comment:
replace plus signs _with_ spaces

Reply:
fixed. would like me to make the same change in the docmentation for the unquote_plus() method?